package-lock.json

```diff
@@ -10410,20 +10409,6 @@
         "url": "https://github.com/sponsors/sindresorhus"
       }
     },
-    "node_modules/xo/node_modules/prettier": {
```

Comment:
I don't know where this has gone or if it is a problem.

Reply:
I guess xo "installs it" and "gets an override. If we agree to use `npx prettier -w` for now, it should be fine. Whenever we actually get to using `xo`, we'll define everything there, including `eslint` config etc.

Reply:
I've asked @SalvatoreZagaria to add a new check that does `npx prettier --check $(echo **/*.ts | tr ' ' '\n' | grep -v "node_modules" | xargs)`.

Reply:
https://github.com/vectorgrp/vector-vscode-vcast/pull/161